testing/src/test/java/io/stargate/it/http/graphql/graphfirst/InsertTest.java

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.it.http.graphql.graphfirst;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+
+import com.datastax.oss.driver.api.core.CqlIdentifier;
+import com.jayway.jsonpath.JsonPath;
+import io.stargate.it.driver.CqlSessionExtension;
+import io.stargate.it.driver.TestKeyspace;
+import io.stargate.it.http.RestUtils;
+import io.stargate.it.storage.StargateConnectionInfo;
+import java.util.UUID;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@ExtendWith(CqlSessionExtension.class)
+public class InsertTest extends GraphqlFirstTestBase {
+
+  private static GraphqlFirstClient CLIENT;
+  private static String NAMESPACE;
+
+  @BeforeAll
+  public static void setup(StargateConnectionInfo cluster, @TestKeyspace CqlIdentifier keyspaceId) {
+    CLIENT =
+        new GraphqlFirstClient(
+            cluster.seedAddress(), RestUtils.getAuthToken(cluster.seedAddress()));
+    NAMESPACE = keyspaceId.asInternal();
+    CLIENT.deploySchema(
+        NAMESPACE,
+        "type User @cql_input {\n"
+            + "  id: ID!\n"
+            + "  name: String\n"
+            + "  username: String\n"
+            + "}\n"
+            + "type Query { user(id: ID!): User }\n"
+            + "type InsertUserResponse @cql_payload {\n"
```

Comment:
shouldn't we consider auto-generating this class similar to `*Input` classes when using (`@cql_input`)?

Reply:
We can't predict the contents though: depending on its intended use, a response payload may wrap a single entity or a list, and may or may not have technical fields. It's even possible to have just the `applied` field if the caller just needs to know if a LWT failed or not, but is not interested in the existing data.

I guess we could encode that information with directive parameters:
```
type User
@cql_payload(name: "InsertUserResponse", entityField: SINGLE, technicalFields: APPLIED)
{
  ...
}
```
But overall I'm not sure if this is an improvement: it's almost as long to type, and it's less clear what's going on.

Reply:
I agree that the current solution is better